Automattic/jetpack
# Create protect-status and protect-models packages

Move protect-models and protect-status classes into their own packages so So that the ability to retrieve the Protect Status (to get threat counts) can be shared with My Jetpack.







Fixes #

## Proposed changes:

* Move Protect status functionality into it's own `protect-status` package.
* Move the models classes into thier own `protect-models` package.
* Update the Protect plugin to use the protect-status and protect-models packages.

### Other information:


## Jetpack product discussion



This is for the _PT: Add value to the active Protect product card on My Jetpack project_: pbNhbs-aP6-p2

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Checkout this branch locally
* Run `jetpack install packages/protect-models` and `jetpack build packages/protect-models`
* Run `jetpack install packages/protect-status` and `jetpack build packages/protect-status`
* From the Protect plugin folder `cd projects/plugins/protect` run `composer require automattic/jetpack-protect-models` and `composer require automattic/jetpack-protect-status`
* Run `jetpack install plugins/protect && jetpack build plugins/protect`
* Go to `/wp-admin/admin.php?page=jetpack-protect` and confirm it loads
